internal/enterprisehooks/installer.go

```
@@ -0,0 +1,647 @@
+			if !info.IsDir() {
+				return fmt.Errorf("enterprise hooks: %s parent is not a directory: %s", label, cur)
+			}
+			if err := validateExistingUserDir(cur, uid, label+" parent"); err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+func validateOptionalExistingUserFile(home, path string, uid int, label string) error {
```

golangci-lint (2.12.2)

[error] func validateOptionalExistingUserFile is unused

(unused)